### ISSUE TITLE
Vendor Dashboard - Order Search

### ISSUE THREAD

**Issue by Tangol** [wcmarketplace/dc-woocommerce-multi-vendor#692]
Please note that the **Vendor Dashboard** is missing a very important feature: **Orders Search**.
Some stores will have hundreds of orders daily and sometimes customers will contact the vendor after days/weeks/months, regarding warranty and other issues.
At the moment it's almost impossible for the vendor to find the customer.
That's why a search using the **Order ID**, **Customer Name** and **Email** is needed.

![search](https://user-images.githubusercontent.com/3336728/108639925-ba59c700-749f-11eb-82ec-36119667aac0.png)

**Comment by moumitahalder** [comment]
Yes, we do have plan to add this feature. @cool-raju please look into this.
